participants/AdventureGo.md

```diff
@@ -0,0 +1,72 @@
+# <AdventureGo>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+（0）.AdventureGo
+（0）.https://filecoinproject.slack.com/team/UPC9BT2BG
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+
+## (1) Brief Project Description
+Offering Filecoin based data, such as mining rankings, blockchain data query, visual chart and other data services.
+（1）Providing Filecoin based data, such as mining rankings, blockchain data query, visual chart and other data services.
+*Add your project's description here.*
+
+## (2) Link to Project UI
+（2）https://www.1475ipfs.com/blockBrowser
+*Please include a link to your project application or UI. It does not need to be super polished or completely functional. We will be reviewing this link periodically to verify that your project is eligible for final rewards.*
+
+## (3) What does your application/UI do?
+（3）Quering Filecoin network data from time to time
+*Give us a brief description of what your application/UI does and how users are intended to interact with it.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+sql text.
+（4）No
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+（5）sql text
+     no
+     yes
+     no
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+（6）300TiB
+*Give us as realistic an estimate as you possibly can at this stage. We understand it's impossible to have perfect information before really starting your project!*
+
+## (7) How are you structuring the data?
+（7）Structured.
```

Comment:
Can you describe a bit more about what type of filecoin data you are going to be storing?